src/averageAges.js

```diff
@@ -37,7 +47,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = (withChildren
+    ? people.filter(
+      ({ name }) => people.some(({ mother }) => mother === name))
+    : people.filter(
+      ({ sex }) => sex === 'f'));
+  const age
+   = women.reduce((sum, person) => sum + (person.died - person.born), 0)
+    / women.length;
+
+  return Math.round(age * 100) / 100;
```

Comment:
аналогічно і в наступному
```suggestion
  return Math.round(age);
```